### ISSUE TITLE
make dynamic fan control default to OFF

### ISSUE THREAD

**Issue by arxo** [aldostools/webMAN-MOD#984]
This is a follow-up to issue #969 .
I ran into the same "fan speed going into jet-engine mode" issue with HFW_4.90.1 on a EU fat PS3.
It drove me totally nuts and took hours until I figured out how to get the speed back to normal.
I would really suggest to have "dynamic fan control" default to OFF in future releases, also enabling it should come with a warning that it might make your fan way more noisy than desired.

**Comment by IAmNadaman** [comment]
I agree with this. It's not like the majority of PS3's need anything more than the system controller's default fan speeds anyway.

The only PS3 models that potentially benefit from increased fan speeds are the ones with 90 nm GPUs, since they have an inherent temperature related design flaw. Not that increasing the fan on these models suddenly make them bullet proof, but it won't hurt either. 

It would be interesting if WMM detected the board ID of your model and only enabled dynamic fan control if it detects you have one of the 90 nm GPU models, and use syscon fan speeds for any newer model (including defaulting to syscon if the user has HEN instead of CFW, since dynamic acts weird in HEN anyway)

**Comment by aldostools** [comment]
Hello friends,
Thank you for your feedback. Unfortunately I disagree with the opinion that only the phat models with 90 nm GPU.
There are many factors that affect the temperature, including the country is in the tropics, the season of the year, ventilation of the area, if used in a room with air conditioning or fans, if paste was replaced recently, etc. 

For instance, my PS3 slim overheats in 10 minutes if I don't use webMAN MOD's dynamic fan control.

In any case, the users should adjust the settings to their preferences after the installation. That's why there is a /setup.ps3

Additionally, the project is open source for people that want to make their own customizations.

**Comment by arxo** [comment]
How about this instead: ask the user upon installation/first start whether to enable dynamic fan control.
That way no one will be caught by surprise by their fans going crazy, and those who require it can quickly turn it on.

**Comment by IAmNadaman** [comment]
> Hello friends, Thank you for your feedback. Unfortunately I disagree with the opinion that only the phat models with 90 nm GPU. There are many factors that affect the temperature, including the country is in the tropics, the season of the year, ventilation of the area, if used in a room with air conditioning or fans, if paste was replaced recently, etc.
> 
> For instance, my PS3 slim overheats in 10 minutes if I don't use webMAN MOD's dynamic fan control.
> 
> In any case, the users should adjust the settings to their preferences after the installation. That's why there is a /setup.ps3
> 
> Additionally, the project is open source for people that want to make their own customizations.

Overheat like, actually triggering the shutdown temperatures? 'Cause that's pretty wild.

**Comment by Joonie86** [comment]
No further feedback received
